public/app/features/integrated-alerting/components/AlertRules/__mocks__/alertRulesStubs.ts

```diff
@@ -1,4 +1,26 @@
 import { AlertRulesListResponseRule } from '../AlertRules.types';
+import { Template } from '../../AlertRuleTemplate/AlertRuleTemplatesTable/AlertRuleTemplatesTable.types';
+
+export const templateStubs: Template[] = [
```

Comment:
If the template stubs are required here can they be imported from the existing stubs?

Reply:
Done